pkg/web/pouchdb-actions.ts

```diff
@@ -0,0 +1,46 @@
+import Pouchdb from "pouchdb";
+
+export interface Note {
+  _id: string;
+  books: BookInterface[];
+}
+
+export interface BookInterface {
+  isbn: string;
+  title: string;
+  author: string;
+  editor: string;
+  price: number;
+  quantity: number;
+  note?: string;
+  date?: string;
+}
+
+export const database = new Pouchdb("notes");
```

Comment:
I think database creation should be part of a `createDB` function. It can use a random ID to make sure an empty database is returned on each invocation.